testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/eap640/DomainAdjuster640.java

```diff
@@ -287,6 +287,10 @@ public void adjustInfinispanStatisticsEnabled(final List<ModelNode> list, final
 
     private Collection<? extends ModelNode> replaceUndertowWithWeb(final PathAddress subsystem) {
         final List<ModelNode> list = new ArrayList<>();
+
+        //mod_cluster depends on the listener capability so we need to remove the subsystem before we can remove undertow
+        list.add(createRemoveOperation(subsystem.getParent().append(SUBSYSTEM, "modcluster")));
```

Comment:
Oh, also note that I am removing it but re-adding back at the bottom. Keeping this in will fail removal of undertow subsystem.